src/components/new-request/search.vue

```diff
@@ -297,6 +298,10 @@ export default class NewSearch extends Vue {
     })
   }
 
+  private activated () {
+    this.scrollTo('namerequest-sbc-header')
+  }
+
```

Comment:
Is this called in the component?

Reply:
no it's a life cycle hook isn't it? I think because of the way the Tabs work it gets triggered? I tried this and it worked, but I've never used it before haha

Reply:
Interesting!! I've never used it, i'll look it up :) Thanks!

Reply:
Hmm, I don't see it here:
https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram

You might want to check where it's called in the order of things. Eg, is it only called once? Has the component been created? Rendered?

Reply:
https://vuejs.org/v2/api/#activated -- not a lifecycle hook I guess

Reply:
well it is a custom lifecycle hook for keep-alive I guess?